commands/output/output.go

```diff
@@ -52,6 +53,15 @@ status: %s
 	// Number of lines of context to print for inline comments
 	contextLineCount = 5
 )
+var default_color = map[string]string{
+	"tbr": "red white bold blink",
+	"pending": "cyan",
+	"submitted": "yellow",
```

Comment:
I feel like the colors for "pending" and "submitted" should be swapped.

I think yellow matches better with "pending" because it indicates that attention is needed. Conversely, cyan makes me think more along the lines of "no action necessary", which would match "submitted".

Reply:
I associate yellow with something old like the paper of an old book becomes yellowish.
There's magenta – which I find pretty aggressive – used for "abandon". How about:
```
	"pending": "magenta",
	"submitted": "cyan",
	"abandon": "yellow",
```

Reply:
That mapping sounds good to me.